profiles/arch/arm64/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Roy Bamford <neddyseagoon@gentoo.org> (12 Mar 2019)
+# Let kde-apps/akonadi use mysql.
```

Comment:
This comment seems backwards - why not look at existing examples and say `dev-db/mysql not keyworded yet` there.

Reply:
How about I change the comment to read
# Allow kde-apps/akonadi to use virtual/mysql 
which is what it really means for the akonadi ebuild.

dev-db/mysql is neither keyworded on arm64 nor required to satisfy virtual/mysql 
arm64 users will actually get dev-db/mariadb

Reply:
Heh, leading hash symbols

Reply:
But what it really means is `allow kde-apps/akonadi to use postgresql or sqlite` while mysql is not available.

Reply:
Ha. No I get it.